opensrp-chw/src/crvs/assets/rule/family_details_remove_member_form_relevance.yml

```diff
@@ -81,5 +81,82 @@ name: step1_marital_status
 description: age_at_death
 priority: 1
 condition: "step1_remove_reason.equals('Died') && step1_age_at_death_calculation > 14"
+actions:
+  - "isRelevant = true"
+---
+name: step1_informant_fields
+description: informant_fields
+priority: 1
+condition: "step1_received_death_certificate('Yes') || step1_death_notification_done('Yes')"
+actions:
```

Comment:
This appears to be missing the comparison operation `.equals()` which breaks the relevance for the `informant_fields` step in the form.
The label currently doesn't show on the form and this requires fixing